digiroad2-oracle/src/main/resources/db/migration/V210__add_new_property_for_bus_stops.sql

```diff
@@ -0,0 +1,34 @@
+INSERT INTO LOCALIZED_STRING(ID, VALUE_FI, CREATED_DATE, CREATED_BY)
+VALUES (primary_key_seq.nextval, 'Pysäkin palvelutaso', CURRENT_TIMESTAMP , 'db_migration_v210');
+
+INSERT INTO PROPERTY(ID, ASSET_TYPE_ID, PROPERTY_TYPE, REQUIRED, CREATED_DATE, CREATED_BY, NAME_LOCALIZED_STRING_ID, PUBLIC_ID, DEFAULT_VALUE)
+VALUES (primary_key_seq.nextval, 10, 'single_choice', '0', CURRENT_TIMESTAMP , 'db_migration_v210', (SELECT max(ID) FROM LOCALIZED_STRING WHERE VALUE_FI = 'Pysäkin palvelutaso'), 'pysakin_palvelutaso', '99');
```

Comment:
Replace the asset_type_id static value by something like this:
`INSERT INTO PROPERTY(ID, ASSET_TYPE_ID, PROPERTY_TYPE, REQUIRED, CREATED_DATE, CREATED_BY, NAME_LOCALIZED_STRING_ID, PUBLIC_ID, DEFAULT_VALUE)
VALUES (primary_key_seq.nextval, (select id from asset_type where name = 'Bussipysäkit'), 'single_choice', '0', CURRENT_TIMESTAMP , 'db_migration_v210', (SELECT max(ID) FROM LOCALIZED_STRING WHERE VALUE_FI = 'Pysäkin palvelutaso'), 'pysakin_palvelutaso', '99');`